packages/preview/cellpress-unofficial/0.1.0/src/lib.typ

```diff
@@ -0,0 +1,81 @@
+// CELL press template.
+
+#let cellpress-red = rgb(169, 59, 69)  // CELL press red
+#let zebra-gray    = rgb(luma(95.25%)) // rgb("#f3f3f3")
+
+// --- [ tables ] --------------------------------------------------------------
+
+#let toprule(color: cellpress-red) = {
+	table.hline(stroke: 0.7pt + color)
+}
+
+#let bottomrule(color: cellpress-red) = {
+	table.hline(stroke: 0.7pt + color)
+}
+
+#let midrule(color: cellpress-red) = {
+	table.hline(stroke: 0.5pt + color)
+}
+
+#let tabsubhdr(content, colspan: 3, color: cellpress-red) = table.header(
```

Comment:
Table subheaders rely on the development version of the typst compiler, or since the table header API seems quite stable `0.14`. Could you document that somewhere? Otherwise people might get confused why this package isn't working for them.